### PR TITLE
Re-generated Network SDK for July release

### PR DESCRIPTION
**Pull request by anton-evseev**
## Description

July release for Network SDK. Based on: https://github.com/Azure/azure-rest-api-specs/pull/3503

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
@number213 please join the Azure org on github [here](aka.ms/azuregithub)

**Comment by dsgouda** [comment]
@MikhailTryakhov is this the official July release for Network?

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 1]
```diff
@@ -7,16 +7,11 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>19.1.0-preview</Version>
+		<Version>19.2.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: @number213 @MikhailTryakhov what are your plans regarding releasing a stable version

**Review by dsgouda — APPROVED** [review]
Looks great for the most part, please address the comments and we are good to merge

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj (hunk 1 above)
Comment (MikhailTryakhov) [reply src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: @dsgouda yes.
Can we also start deployment process to other SDKs as well to be synced

Comment (dsgouda) [reply src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: Not sure what that means, you will have to talk to individual SDK teams regarding this

**Pull request merged**